src/NServiceBus.TransportTests/NServiceBusTransportTest.cs

```diff
@@ -171,7 +171,7 @@ protected void OnTestTimeout(Action onTimeoutAction)
         {
             testCancellationTokenSource = new CancellationTokenSource();
 
-            testCancellationTokenSource.CancelAfter(TimeSpan.FromSeconds(30));
+            testCancellationTokenSource.CancelAfter(Debugger.IsAttached ? TimeSpan.FromMinutes(5) :TimeSpan.FromSeconds(30));
```

Comment:
AcceptanceTests run indefinitely when the debugger is attached. Should this run indefinitely as well or would 5 min be a good default for the acceptance tests as well?